docs/language/operators/README.md

```diff
@@ -9,9 +9,11 @@ and appear as the components of a dataflow pipeline.
 * [combine](combine.md) - combine parallel paths into a single output
 * [cut](cut.md) - extract subsets of record fields into new records
 * [drop](drop.md) - drop fields from record values
+* [file](file.md) - source data from a file
 * [from](from.md) - source data from pools, files, or URIs
 * [fork](fork.md) - copy values to parallel paths
 * [fuse](fuse.md) - coerce all input values into a merged type
+* [get](get.md) - source data from a URI
```

Comment:
@nwt: Ok. I'm fine with taking this suggestion so we'll be linking directly from `get`/`file` to `from`. However, I'm going to leave the `get.md` and `file.md` in place with their redirecting contents because that's the only way I see to still get autogenerated entries for them in the sidebar.